src/jit/importer.cpp

```diff
@@ -6462,6 +6463,11 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         call->gtCall.callSig  = new (this, CMK_CorSig) CORINFO_SIG_INFO;
         *call->gtCall.callSig = calliSig;
 #endif // DEBUG
+
+        if (IsTargetAbi(CORINFO_CORERT_ABI))
```

Comment:
See a few lines below: `case CORINFO_VIRTUALCALL_LDVIRTFTN`
And then: `GenTreePtr fptr = impImportLdvirtftn(thisPtr, pResolvedToken, callInfo)`
fptr is the target address that will be called, and could be a fat function pointer.